src/components/Output/custom-els/RangeInput/index.ts

```diff
@@ -0,0 +1,94 @@
+import './styles.css';
+
+const EVENTS = [
+  'input',
+  'change',
+  'focus',
+  'blur',
+];
+
+const PROPERTIES = [
+  'name',
+  'value',
+  'min',
+  'max',
+  'step',
+];
+
+interface RangeInputElement {
+  value: string;
+  min?: string;
+  max?: string;
+  step?: string;
+  precision?: number | string;
+}
+
+class RangeInputElement extends HTMLElement {
+  private _input = document.createElement('input');
+  private _valueDisplayWrapper = document.createElement('aside');
+  private _valueDisplay = document.createElement('span');
+
+  constructor() {
+    super();
+    this._input.type = 'range';
+    this._update = this._update.bind(this);
+    this._handleEvent = this._handleEvent.bind(this);
```

Comment:
Use `@bind` for these two.

Reply:
didn't think we were using that in the web components. added.